samTemplates/amazon-connect-interactive-messages-example-lex-v2/interactive-messaging-lex-codehook/util/user_input_handler.js

```diff
@@ -0,0 +1,114 @@
+/* user_input_hander.js HANDLES INPUTS PROVIDED BY THE USER TO THE CHAT */
+
+const {
+  formElicitSlotWithTemplateResponse,
+  formTerminalResponse,
+  formElicitIntentResponse,
+} = require("./response_handler");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We detected that you are importing a module inside a function, which is known as lazy loading. If modules are imported within a function, it might prevent other requests from being handled at a more critical time. We recommend that you load all modules at the beginning of each file, before and outside of any functions.